gpu/gpu4pyscf/mcscf/df.py

```diff
@@ -0,0 +1,518 @@
+#!/usr/bin/env python
+# Copyright 2014-2020 The PySCF Developers. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Author: Qiming Sun <osirpt.sun@gmail.com>
+#
+
+
+import ctypes
+from functools import reduce
+import numpy
+from pyscf import lib
+from pyscf.lib import logger
+from pyscf.ao2mo import _ao2mo
+from pyscf.mcscf.casci import CASCI
+from pyscf import df
+from gpu4pyscf.lib.utils import patch_cpu_kernel
+from mrh.my_pyscf.gpu import libgpu
+
+# Setting DEBUG = True will execute both CPU (original) and GPU (new) paths checking for consistency 
```

Comment:
Is this true? Because the variable `DEBUG` doesn't appear in this file after line 34.